litmus-portal/frontend/src/components/WorkflowCluster/index.tsx

```diff
@@ -24,12 +25,14 @@ function Check() {
 */
 const WorkflowCluster = () => {
   const classes = useStyles();
-  const [value, setValue] = React.useState('female');
+  const [value, setValue] = React.useState('Experiment');
 
   const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     setValue((event.target as HTMLInputElement).value);
   };
-
+  const handleClick=()=>{
```

Comment:
Avoid empty functions, if you are not doing any on Click handler better just do a () => {} at the button itself or rather not keep it at all